contrib/kafka/pkg/adapter/adapter.go

```diff
@@ -77,8 +77,9 @@ func (a *Adapter) ConsumeClaim(sess sarama.ConsumerGroupSession, claim sarama.Co
 	logger := logging.FromContext(context.TODO())
 
 	for msg := range claim.Messages() {
-		logger.Info("Received: ", zap.Any("value", string(msg.Value)))
-
+		logger.Info("Received: ", zap.String("topic:", string(msg.Topic)),
```

Comment:
No need for the casts here. `msg.Topic`, `msg.Partition`, and `msg.Offset` are already the expected types.

Reply:
Ok, Partition variable are int32's and Offset is int64.  Would you consider it acceptable to use zap.Int32() and zap.Int64() for the respective variables instead of the cast?

Reply:
Yep that's what I'd prefer!